nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -533,6 +533,29 @@ impl<'a> BlueprintBuilder<'a> {
             .with_context(|| {
                 format!("failed to construct SledEditor for sled {sled_id}")
             })?;
+
+            // Apply fixes for omicron-7229 to all active sleds: If any zones
```

Comment:
nit: the other PR uses "#7229"; could be consistent

Reply:
thanks, fixed in [d1f250f](https://github.com/oxidecomputer/omicron/pull/7450/commits/d1f250f472887cc27473504ce70a8246258c6025)